Quadriga/src/main/java/edu/asu/spring/quadriga/web/workspace/ConceptCollectionWorkspaceController.java

```
@@ -32,270 +32,248 @@
-						flag=1;
-					}
-				} catch (Exception e) {
-					logger.error("Issue while accessing DB",e);
-				}
-			}
-		}
-		if(flag==0){
-			model.addAttribute("success", 1);
-		}else{
-			model.addAttribute("success", 0);
-		}
-		List<IWorkspaceConceptCollection> conceptCollectionList = null;
-		try {
-			conceptCollectionList = workspaceCCManager.listWorkspaceCC(workspaceId, userId);
-		} catch (QuadrigaStorageException e) {
-			logger.error("Issue while accessing DB",e);
-		}
-		if(conceptCollectionList == null){
-			logger.info("conceptCollectionList list is empty buddy");
-		}
-		model.addAttribute("conceptCollectionList", conceptCollectionList);
-		IWorkSpace workspace = wsManager.getWorkspaceDetails(workspaceId,userId);
-		model.addAttribute("workspacedetails", workspace);
-		model.addAttribute("workspaceId", workspaceId);
-		return "auth/workbench/workspace/conceptcollections";
-	}
+            List<IConceptCollection> conceptCollectionList = null;
+            try {
+                conceptCollectionList = workspaceCCManager.getNonAssociatedWorkspaceConcepts(workspaceId, userId);
+            } catch (QuadrigaStorageException e) {
+                logger.error("Issue while accessing DB", e);
+            }
+            if (conceptCollectionList == null) {
+                logger.info("conceptCollectionList list is empty");
+            }
+            Iterator<IConceptCollection> I = conceptCollectionList.iterator();
+            while (I.hasNext()) {
+                IConceptCollection con = I.next();
+                logger.info(" " + con.getConceptCollectionName());
+            }
+            model.addAttribute("conceptCollectionList", conceptCollectionList);
+            IWorkSpace workspace = wsManager.getWorkspaceDetails(workspaceId, userId);
+            model.addAttribute("workspacedetails", workspace);
+            model.addAttribute("workspaceId", workspaceId);
+            model.addAttribute("userId", userId);
+        } catch (Exception e) {

```

needs to be fixed (no catch all exceptions)